src/gettingstarted/own-code/app-configuration.md

```diff
@@ -0,0 +1,106 @@
+
+    > **Note**  If you are trying to to deploy [microservices](/configuration/app/multi-app.md#example-of-a-micro-service-multi-app), the only constraint is that each of these application names must be unique.
+
+* `type`: The [type](/configuration/app/type.md) attribute in `.platform.app.yaml` sets the container base image for the application, and sets the primary language. In general, `type` should have the form
+
+  ```yaml
+  type: <runtime>:<version>
+  ```
+
+  Set `version` to one supported by Platform.sh, which you can find below as well as in the documentation for each language:
+
+<div>
+  <table id="runtimeTable" border="1">
+  <tbody></tbody>
+  </table>
+</div>
+
+<script>
+makeImagesTable("runtimes", "supported", "runtimeTable");
+</script>
+
+* `disk`: The [disk](/configuration/app/storage.md) attribute defines that amount of persistent storage you need to have available for your application, and requires a minimum value of 256 MB.
+
+* `mounts` (Optional): Configuring mounts are not required, unless part of your application requires write-access. By default, Platform.sh provided a *read-only* filesystem for your projects, so that you can be confident in the health and security of your application once it has deployed. If your application requires writable storage to function properties (i.e., saving files; mounts should not contain code) it can be defined:
+
+  If your application requires writable storage to function properly, it can be defined:
+
+  ```yaml
+  mounts:
+    'web/uploads':
+        source: local
+        source_path: uploads
+  ```
+
+  In this case, the application will be able to write to a mount that is visible in the `/app/web/uploads` directory of the application container, and which has a local source at `/mnt/uploads`. Consult the [mounts documentation](/configuration/app/storage.md#mounts) for a more thorough discussion of how these attributes should be written.
+
+* `web`: The `web` key configures the web server through a single web instance container running a single Nginx server process, behind which runs your application.
+
+    * `commands`: Defines the [command](/configuration/app/web.md#locations) to actually launch the application. The `start` key launches your application.
+    * `locations`: Allows you to control how the application container responds to incoming requests at a very fine-grained level. The simplest possible [locations](/configuration/app/web.md#locations) configuration is one that simply passes all requests on to your application unconditionally:
+
+      ```yaml
+      web:
+      locations:
+          '/':
+              passthru: true
+      ```
```

Comment:
PHP is the odd-ball-out here, and also are largest business.  How should we handle describing that `web.commands.start` is mostly pointless on PHP, but `passthru` needs to be a script name?